src/components/fields/MultipleChoice.js

```diff
@@ -145,41 +146,52 @@ class MultipleChoice extends AskField {
     return { options: selectedOptions }
   }
 
-  render () {
+  getCharIndex(i) {
+    return String.fromCharCode(65 + i);
+  }
+
+  render() {
     return (
       <div>
         <fieldset
-          style={styles.base}>
-          <legend style={styles.accesibleLegend}>{this.props.title}</legend>
-          {this.props.options && !!this.props.options.length
-            ? <div>
+          style={ styles.base }>
+          <legend style={ styles.accesibleLegend }>{ this.props.title }</legend>
+          { this.props.options && !!this.props.options.length ?
+              <div style={ styles.optionsWrapper }>
 
                 {this.getOptions()}
 
                 {
-                  this.props.otherAllowed
-                  ? <div>
-                    <label
-                      // onMouseOver={ this.onHover.bind(this, i) }
-                      style={this.getOtherStyle()}
-                      key={this.props.options.length}
-                      ><input
-                        style={styles.optionCheck}
-                        onChange={this.onOtherClick.bind(this)}
-                        tabindex='0'
-                        name={!this.props.multipleChoice ? this.props.title : false}
-                        type={this.props.multipleChoice ? 'checkbox' : 'radio'}
-                        key={this.props.options.length}
-                      />
-                        Other
-                      <input type='text' onChange={this.onOtherChange.bind(this)} style={styles.otherInput} />
-                    </label>
-                  </div>
+                  this.props.otherAllowed ?
+                    <div>
+                      <label
+                          //onMouseOver={ this.onHover.bind(this, i) }
+                          style={ this.getOtherStyle() }
+                          key={ this.props.options.length }
+                        ><input
+                            style={ styles.optionCheck }
+                            onClick={ this.onOtherClick.bind(this) }
+                            tabindex="0"
+                            name={ !this.props.multipleChoice ? this.props.title : false }
```

Comment:
probably not a big deal, but if you remove the leading `!` on the ternary, you could just reverse the terms and it'd be a bit simpler